README.md

```diff
@@ -22,7 +22,10 @@ git clone git@github.com:wagtail/docker-wagtail-develop.git wagtail-dev
 cd wagtail-dev/
 # 4. Run the setup script. This will check out all the dependency repos.
 ./setup.sh
-# 5. Build the containers
+# 5. Configure your app to use PostgreSQL via the DATABASE_URL from docker-compose.yml
+cp bakerydemo/bakerydemo/settings/local.py.docker-compose-example bakerydemo/bakerydemo/settings/local.py
```

Comment:
Gah, tech fail. The suggestion was to move these into `setup.sh` so the setup part has fewer steps one needs to copy/paste

Reply:
@zerolab moved the database configuration into the setup.sh script that checks out the repositories